src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRailArtworks.tsx

```diff
@@ -24,32 +26,40 @@ const AuctionArtworksRailArtworks: React.FC<AuctionArtworksRailArtworksProps> =
   const { contextPageOwnerType } = useAnalyticsContext()
   const contextModule = tabTypeToContextModuleMap[tabType]
 
+  const AUCTION_ARTWORKS_RAIL_HEIGHT = 233
+
+  if (sale.artworksConnection.edges.length === 0) {
+    return null
+  }
+
```

Comment:
This logic is what hides the Artwork images. If we do not have any artworks to display we return null.

Reply:
Ahh I see, it hides _all_ the images -- i was thinking it just hid unpublised ones, my confusion.